data/wi/legislature/Romaine-Quinn-db7ed090-5b67-4ebc-9862-1cfa408f5f0c.yml

```diff
@@ -0,0 +1,32 @@
+id: ocd-person/db7ed090-5b67-4ebc-9862-1cfa408f5f0c
+name: Romaine Quinn
+given_name: Romaine
+family_name: Quinn
+email: Sen.Quinn@legis.wisconsin.gov
```

Comment:
@bfossen-ce this person was in assembly district 75 from 2015 until January 3, 2021.